content/docs/web-components.md

```diff
@@ -20,14 +20,14 @@ class HelloMessage extends React.Component {
 }
 ```
 
-> Note:
+> Pamiętaj:
 >
-> Web Components often expose an imperative API. For instance, a `video` Web Component might expose `play()` and `pause()` functions. To access the imperative APIs of a Web Component, you will need to use a ref to interact with the DOM node directly. If you are using third-party Web Components, the best solution is to write a React component that behaves as a wrapper for your Web Component.
+> Komponenty sieciowe często udostępniają imperatywne API. Na przykład komponent sieciowy `video` może udostępniac funkcje `play()` i `pause()`. Aby skorzystać z imperatywnego API komponentu sieciowego, możesz potrzebować użyć referencji, aby bezpośrednio nawiązać kontakt z węzłem DOMu. Jeśli korzystasz z komponentów sieciowych innych osób, najlepszym rozwiązanie będzie utworzyć komponent React służący jako nakładka dla komponentu sieciowego.
```

Comment:
```suggestion
> Komponenty sieciowe często udostępniają imperatywne API. Na przykład, komponent sieciowy `video` może udostępniać funkcje `play()` i `pause()`. Do skorzystania z imperatywnego API komponentu sieciowego możesz potrzebować referencji (`ref`), aby nawiązać bezpośredni kontakt z węzłem drzewa DOM. Jeśli korzystasz z komponentów sieciowych innych osób, najlepszym rozwiązanie będzie utworzenie komponentu reactowego opakowującego komponent sieciowy.
```

Reply:
- Po `Na przykład` te przecinek średnio pasuje.
- `Do` w tym wypadku nie pasuje.
- Pozostawiłem `utworzyć komponent służący jako nakładka`, po `opakowującego` brzmi gorzej.

Reply:
Po `Na przykład` powinien być przecinek, ewentualnie dwukropek. W tym przypadku jest to wtrącenie.
Co do `utworzyć`: po "najlepszym rozwiązaniem będzie..." musi stanąć rzeczownik, a nie bezokolicznik.